resources/tentacle.rb

```diff
@@ -39,3 +39,4 @@
 attribute :environment, kind_of: [String, Array], default: node.chef_environment
 attribute :tenants, kind_of: Array, default: nil
 attribute :tenant_tags, kind_of: Array, default: nil
+attribute :public_dns, kind_of: String, default: node['hostname'].to_s
```

Comment:
I do not think you need the `to_s` Also wouldnt this be better to use `node['fqdn']`